Signal/src/ViewControllers/AddToBlockListViewController.m

```diff
@@ -332,6 +375,113 @@ - (BOOL)textFieldShouldReturn:(UITextField *)textField
     return NO;
 }
 
+#pragma mark - UITableViewDataSource
+
+- (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView
+{
+    return 1;
+}
+
+- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section
+{
+    return (NSInteger)self.contacts.count;
+}
+
+- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath
+{
+    Contact *contact = self.contacts[(NSUInteger)indexPath.item];
+
+    ContactTableViewCell *cell = [_contactsTableView cellForRowAtIndexPath:indexPath];
+    if (!cell) {
+        cell = [ContactTableViewCell new];
+    }
+    [cell configureWithContact:contact contactsManager:self.contactsManager];
```

Comment:
Yup, so a few things.

* I'm already filtering blocked users from this list in https://github.com/WhisperSystems/Signal-iOS/pull/1928 .  The other option was to show them in this list but with a checkmark, but I felt it was slightly better to filter them?
* I have an open trello card to pop the "add block" view after a successful block.  Most users will only want to add a single block at a time, and this will - as you point out - give them the context to see how their block list has changed.